tests/conftest.py

```diff
@@ -206,6 +207,10 @@ def rand_one_dut_hostname(request):
         dut_hostnames = random.sample(dut_hostnames, 1)
     return dut_hostnames[0]
 
+@pytest.fixture(scope="module", autouse=True)
+def set_selected_tor_active(duthosts, rand_one_dut_hostname):
```

Comment:
As per our discussion yesterday, for now it may be better to always set the upper ToR to active at the start of the test

Reply:
I remembered that some guys suggested we keep random selection. Did I misunderstand something?

Reply:
I think we had considered making it random, and eventually decided to keep it consistent for now. Making it random is something we can discuss in the future, but right now it would introduced too much complexity for the tests.